Microsoft.Toolkit.Uwp.UI/Media/CanvasBrushBase.cs

```diff
@@ -77,6 +79,12 @@ protected override void OnConnected()
             }
         }
 
+        private void CanvasDevice_DeviceLost(CanvasDevice sender, object args)
+        {
+            OnConnected();
+            OnDisconnected();
```

Comment:
Should these be flipped?  Don't we need to re-connect the brush on devicelost?  I'll see if I can pull these changes later and test it.  But this looks much better!

Reply:
I tested them flipped and I had the same result.